src/Microsoft.ML.TensorFlow/doc.xml

```diff
@@ -60,49 +60,7 @@
         </a>.
 
       </remarks>
-      <example>
```

Comment:
>example [](start = 7, length = 7)

Should we add an example using the new API instead?

Reply:
Do you plan to add it to Microsoft.ML.Samples in this PR, or later?

---
In reply to: [236427095](https://github.com/dotnet/machinelearning/pull/1714#discussion_r236427095) [](ancestors = 236427095)

Reply:
I plan to add it in a separate PR. 

Created issue #1715 to track that work.  Also, such samples should be in Microsoft.ML.Samples  (instead of embedding them inside doc.xml)



---
In reply to: [236472542](https://github.com/dotnet/machinelearning/pull/1714#discussion_r236472542) [](ancestors = 236472542,236427095)